_config.yml

```diff
@@ -12,8 +12,6 @@ url: 'https://wott.io'
 title: Web of Trusted Things
 description: WoTT is a free, automated, open-source public key infrastructure for connected hardware
 cover: assets/images/blog-cover.jpg
-logo: assets/images/blog-icon.png
```

Comment:
Isn't this supposed to be the favicon?

Reply:
@vpetersson this is unused code

Reply:
Ok, let's create another issue for that then.